assets/operator/ocp-ppc64le/eap/templates/eap74-sso-s2i.json

```
@@ -614,126 +640,128 @@
 			"name": "APPLICATION_NAME",
 			"displayName": "Application Name",
 			"description": "The name for the application.",
-			"value": "datavirt-app",
+			"value": "eap-app",
+			"required": true
+		},
+		{
+			"name": "EAP_IMAGE_NAME",
+			"displayName": "EAP Image Name",
+			"description": "Name of the EAP image to use, example: jboss-eap74-openjdk11-openshift:latest",
+			"value": "jboss-eap74-openjdk11-openshift:7.4.0",
 			"required": true
 		},
 		{
-			"name": "CONFIGURATION_NAME",
-			"displayName": "Configuration Secret Name",
-			"description": "The name of the secret containing configuration properties for the data sources.",
-			"value": "datavirt-app-config",
+			"name": "EAP_RUNTIME_IMAGE_NAME",
+			"displayName": "EAP Runtime Image Name",
+			"description": "Name of the EAP image to use, example: jboss-eap74-openjdk11-runtime-openshift:latest",
+			"value": "jboss-eap74-openjdk11-runtime-openshift:7.4.0",
+			"required": true
+		},
+		{
+			"name": "SSO_IMAGE_NAME",
+			"displayName": "SSO Image Name",
+			"description": "Name of the SSO image to use, example: sso74-openshift-rhel8:latest",
+			"value": "sso74-openshift-rhel8:latest",
```

https://github.com/jboss-container-images/jboss-eap-openshift-templates/pull/49 will fix this, since SSO 7.4 is no longer supported and is simultaneously being removed:

```suggestion
			"description": "Name of the SSO image to use, example: sso75-openshift-rhel8:latest",
			"value": "sso75-openshift-rhel8:latest",
```